crates/shared/src/price_estimation/http.rs

```diff
@@ -129,7 +129,7 @@ impl HttpPriceEstimator {
                     token: self.native_token,
                 },
                 is_liquidity_order: false,
-                mandatory: true,
+                mandatory: false,
```

Comment:
Why change this here?

Reply:
Because I've deprecated the field, so it should always be set to `false`.